jest.config.js

```diff
@@ -107,7 +118,7 @@ module.exports = {
   // restoreMocks: false,
 
   // The root directory that Jest should scan for tests and modules within
-  // rootDir: null,
+  // rootDir: "./packages/",
```

Comment:
o_O this is commented out.

Reply:
Supposed to be – I was playing with the option, but we don't need it.

Reply:
That said, I can certainly put the comment back at the default, or remove all comments. I didn't mean to change this line.

Reply:
Update: it's now back to the default value in the comment. There's an argument that we should consider removing the unused options in this file, but I find it's nice to have inline documentation on the defaults for each. I could, however, be easily swayed.

Reply:
Yeah I'm fine either way. No real preference. Could remove and always look up the options later.